### PR TITLE
GameDB: Enter the Matrix changes

### PR DESCRIPTION
**Pull request by atomic83GitHub**
This PR add some explaination about what the EETiming hack does in "Enter the Matrix" and add a missing serial with corresponding fixes to it.

Tested by atomic83github

### CODE REVIEW

**Comment by lightningterror** [comment]
Can the forum compatibility be updated first? Will merge this once those are done so we don't forget.

**Comment by bositman** [comment]
The game compat status is a snapshot from the compatibility forum with some tool we had, it's bad practice to update games by hand

**Comment by atomic83GitHub** [comment]
So I create the forum report and close that PR?

**Comment by lightningterror** [comment]
Just leave out the compatibility changes.

**Comment by bositman** [comment]
And also make a compatibility report on the forum so we get the proper status next time
https://forums.pcsx2.net/Thread-Enter-the-Matrix-SLUS-20454-U--28559 (or whatever game region you got)

File: bin/GameIndex.dbf [hunk 1]
```diff
@@ -9344,8 +9344,8 @@ Region = PAL-M3
 Serial = SLES-51203
 Name   = Enter the Matrix
 Region = PAL-M5
-Compat = 5
-EETimingHack = 1
+Compat = 4
```
Comment (lightningterror) [inline bin/GameIndex.dbf]: This one too, also don't forget to update commit description.

**Pull request merged**